pulp_2to3_migration/app/migration.py

```diff
@@ -321,15 +321,18 @@ def create_repo_version(progress_rv, pulp2_repo, pulp3_remote=None):
         pulp3_remote(remote): a pulp3 remote
     """
 
+    pulp3_repo = pulp2_repo.pulp3_repository
     if pulp3_remote:
         pulp2_repo.pulp3_repository_remote = pulp3_remote
         pulp2_repo.save()
 
+        pulp3_repo.remote = pulp3_remote
+        pulp3_repo.save()
```

Comment:
I am not sure this would work for all plugins. In Pulp 3 each plugin needs to implement this association for it to be available. For `pulp_deb` I only happened to stumble across this recently (See https://github.com/pulp/pulp_deb/pull/261) but it is not yet released.

Reply:
@quba42 This appears to be released now, is that correct?

Reply:
@quba42 , this is planned to go into 0.12 release which will be compatible with pulpcore 3.12-3.14.
This release will no longer compatible with Katello. They are staying on 0.11.z and won't go beyond that.

Reply:
The pulp_deb change was released with version 2.12.0 which is compatible with `pulpcore>=3.12,<3.14`, so that should line up nicely.